doc/nrf/releases_and_maturity/releases/release-notes-3.2.0.rst

```diff
@@ -0,0 +1,1352 @@
+  * The :ref:`Audio application API documentation <audio_api>` page.
+  * The :ref:`config_audio_app_options` page.
+  * The API documentation in the header files listed on the :ref:`audio_api` page.
+  * Ability to connect by address as a unicast client.
+
+* Updated:
+
+  * The unicast client (gateway) application has been rewritten to support N channels.
+  * The unicast client (gateway) application now checks if a server has a resolvable address.
+    If this has not been resolved, the discovery process will start in the identity resolved callback.
+  * The power measurements to be disabled by default in the default debug versions of the applications.
+    To enable power measurements, see :ref:`nrf53_audio_app_configuration_power_measurements`.
+  * The audio application targeting the :zephyr:board:`nrf5340dk` to use pins **P1.5** to **P1.9** for the I2S interface instead of **P0.13** to **P0.17**.
+    This change was made to avoid conflicts with the onboard peripherals on the nRF5340 DK.
+  * The documentation pages with information about the :ref:`SD card playback module <nrf53_audio_app_overview_architecture_sd_card_playback>` and :ref:`how to enable it <nrf53_audio_app_configuration_sd_card_playback>`.
+  * The API documentation in the header files listed on the :ref:`audio_api` page.
+
+* Removed the LC3 QDID from the :ref:`nrf53_audio_feature_support` page.
+  The QDID is now listed in the `nRF5340 Bluetooth DNs and QDIDs Compatibility Matrix`_.
+
+nRF Desktop
+-----------
+
+* Updated:
+
+  * The memory layouts for the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target to make more space for the application code.
+    This change in the partition map of every nRF54LM20 configuration is a breaking change and cannot be performed using DFU.
+    As a result, the DFU procedure will fail if you attempt to upgrade the application firmware based on one of the |NCS| v3.1 releases.
+  * The application and MCUboot configurations for the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target to use the CRACEN hardware crypto driver instead of the Oberon software crypto driver.
+    The application image signature is verified with the CRACEN hardware peripheral.
+  * The MCUboot configurations for the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target to use the KMU-based key storage.
+    The public key used by MCUboot for validating the application image is securely stored in the KMU hardware peripheral.
+    To simplify the programming procedure, the application is configured to use the automatic KMU provisioning.
+    The KMU provisioning is performed by the west runner as a part of the ``west flash`` command when the ``--erase`` or ``--recover`` flag is used.
+  * Application configurations to avoid using the deprecated Kconfig options :ref:`CONFIG_DESKTOP_HID_REPORT_EXPIRATION <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_HID_EVENT_QUEUE_SIZE <config_desktop_app_options>`.
```

Comment:
```suggestion
  * Application configurations to avoid using the deprecated Kconfig options :option:`CONFIG_DESKTOP_HID_REPORT_EXPIRATION` and :option:`CONFIG_DESKTOP_HID_EVENT_QUEUE_SIZE`.
```